File: js/aoc.js
const arr = require("./input.json")

function day1(arr){
  for(a=0;a<arr.length;a++){
    if(arr.includes(2020-arr[a])){
      return (2020-arr[a])*arr[a]
    }
  }
}

function day2(arr){
  for(a=0;a<arr.length;a++){
    for(b=1;b<arr.length;b++)
    if(arr.includes(2020-arr[a]-arr[b])){
      return (2020-arr[a]-arr[b])*arr[a]*arr[b]
    }
  }
}
console.log(day1(arr))
console.log(day2(arr))

File: py/aoc.py
with open("input.txt", "r") as file:
    list = list(map(lambda x: int(x), file.readlines()))

def day1(list):
	for i in list:
		if 2020-i in list:
			return (2020-i)*i
			
def day2(list):
	for i in list:
		for a in list:
			if 2020-i-a in list:
				return((2020-i-a)*a*i)
			
print(f"day 1, solution = {day1(list)}")
print(f"day 2, solution = {day2(list)}")